test/wikidata_ids_decorator_test.rb

```diff
@@ -4,4 +4,18 @@
   it 'has a version number' do
     ::WikidataIdsDecorator::VERSION.wont_be_nil
   end
+
+  describe 'Wikipedia page with links' do
+    it 'adds a data-wikidata attribute to links' do
+      body = <<-HTML
+<div id="bodyContent">
+  <a href="https://en.wikipedia.org/wiki/Douglas_Adams" title="Douglas Adams">Douglas Adams</a>
+</div>
```

Comment:
I think this falls into a recurring trap of making the test data function differently from what real data looks like — e.g. here we have absolute URLs, whereas wikipedia uses relative ones, and by not having any links outside `#bodyContent`, we aren't testing that those don't get included.  It's also not obvious here what will happen with edge cases like anchored links (e.g. when scraping https://en.wikipedia.org/wiki/Life,_the_Universe_and_Everything that it does the right thing with `<a href="/wiki/OCLC#Identifiers_and_linked_data" title="OCLC">`) or with links that are redirects (from the same page, `<a href="/wiki/Vogons" class="mw-redirect" title="Vogons">`)

I'm also not convinced we should be doing live lookups in the test, so if we're going to have to stub or VCR record the Wikidata lookup, we should probably do likewise for a real Wikipedia page too (and probably more than one, as we'll want to test that it works for more than en.wikipedia)